src/sign.jl

```diff
@@ -26,7 +26,7 @@ end
 function sign_aws2!(r::AWSRequest, t)
 
     query = Dict{AbstractString,AbstractString}()
-    for elem in split(r[:content], '&', keep=false)
+    for elem in split(r[:content], '&', keepempty=false)
```

Comment:
The default is `false` so don't really need this but if you're being explicit then maybe keep it